sdk/formrecognizer/azure-ai-formrecognizer/tests/test_invoice_async.py

```diff
@@ -260,31 +260,6 @@ def callback(raw_response, _, headers):
 
         self.assertFormPagesTransformCorrect(returned_model.pages, read_results, page_results)
 
-    @FormRecognizerPreparer()
-    @GlobalClientPreparer()
-    async def test_invoice_pdf(self, client):
```

Comment:
Just wondering why this test was removed?

Reply:
Sorry should have called that out - it was a duplicate test.

Reply:
Ah got it. Thanks for the info! :)